start-testnet-reth-miner-node.sh

```
@@ -0,0 +1,51 @@
+# !/bin/sh
```

Shellcheck (0.10.0)

[error] Remove spaces between # and ! in the shebang.

(SC1115)